client/components/BikePopup.tsx

```diff
@@ -0,0 +1,90 @@
+import { StyleSheet, Alert, View, Text, Pressable, Button } from 'react-native';
+import { Callout } from 'react-native-maps';
+import { Entypo } from '@expo/vector-icons'; 
```

Comment:
There is also Ionicons if you need more options. https://aboutreact.com/react-native-ionicons/